frontend-react/src/pages/error/legacy-content/Generic.tsx

```diff
@@ -0,0 +1,74 @@
+import Helmet from "react-helmet";
+import React from "react";
+import { Alert } from "@trussworks/react-uswds";
+
+import {
+    GENERIC_ERROR_PAGE_CONFIG,
+    GENERIC_ERROR_STRING,
+} from "../../../content/error/ErrorMessages";
+import { ErrorDisplayConfig, ErrorPageContentConfig } from "../ErrorPage";
+
+export const GenericMessage = ({ message }: { message?: string }) => {
+    return (
+        <Alert type="error">{message ? message : GENERIC_ERROR_STRING}</Alert>
+    );
+};
+
+export const GenericPage = ({
+    config = GENERIC_ERROR_PAGE_CONFIG as ErrorPageContentConfig,
+}: {
+    config?: ErrorPageContentConfig;
+}) => {
+    return (
+        <>
+            <Helmet>
+                <title>Error | {process.env.REACT_APP_TITLE}</title>
+            </Helmet>
+            <div
+                data-testid={"error-page-wrapper"}
+                className="usa-section padding-top-6"
+            >
+                <div className="grid-container">
+                    <div className="grid-row grid-gap">
+                        <div className="usa-prose">
+                            <h1>{config.header}</h1>
+                            <p>{config.paragraph}</p>
+                            <div className="margin-y-5">
+                                <ul className="usa-button-group">
+                                    <li className="usa-button-group__item">
+                                        <a href="./" className="usa-button">
+                                            Visit homepage
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </>
+    );
+};
+
+export interface GenericErrorProps {
+    displayAsPage?: boolean;
+    displayConfig?: ErrorDisplayConfig;
+}
+export const GenericError = ({
+    displayAsPage,
+    displayConfig,
+}: GenericErrorProps) => {
+    if (!displayConfig) {
+        // For back-compat with older uses
+        // TODO: Remove when we stop using GenericError outside of RSErrorBoundary
+        return displayAsPage ? <GenericPage /> : <GenericMessage />;
+    } else {
+        // For use with RSNetworkError
+        // Error message/page configs are designed in `/src/content/error/ErrorMessages.ts`
+        return typeof displayConfig === "string" ? (
```

Comment:
Even this is a little shoddy, but the reason I'm against the notion is because I'd ideally love to toss aside the notion of `message` and `page` entirely. _Where_ you wrap the components determines how the UI renders, not the size/shape of the error UI itself. So even displaying a "page" as the fallback for one component on a page with many islands would result in a small rectangular error "page" in the middle of all the other components.

The way its set up now is strictly for hydrating different versions of GenericError's UI, but even _that_ is something I think we should move away from. Before we make any more visual changes to our Error UIs I want design to mock up how they envision error handling with the tools/knowledge we've gained during this process.

Reply:
An idea bouncing around my head right now is "well, if it's a table that fails to load, we should still show an empty, errored-out table, right?" -- you brought this up I think on a former PR. That's the direction I'd like to go with error displaying, but again, should def get design on it.

Reply:
How I've solved this for now is changing the wording/names:

`ErrorDisplayMessage` is now a union type of `ParagraphWithTitle | string` -- this indicates that we have to unique ways of displaying a message to the user, either a single line as is often the use case for an alert, OR a title/paragraph model.

Then, `GenericError` is now `BasicErrorDisplay`, which will parse between `StringErrorDisplay` and `ParagraphErrorDisplay`